src/gsAssembler/gsAssembler.h

```diff
@@ -454,9 +454,8 @@ class gsAssembler
     {
         for (index_t np = 0; np < m_pde_ptr->domain().nPatches(); ++np)
         {
-            ElementVisitor curVisitor = visitor;
             //Assemble (fill m_matrix and m_rhs) on patch np
-            apply(curVisitor, np);
+            apply(visitor, np);
```

Comment:
Hi Alex, do you need this change or is it just to avoid the extra copy ?
the copy is needed for paralellization...

Reply:
No, I don't need it. I was just wondering: in the next function , push a nontrivial boundary visitor, there is no copy. Now I remember that we did it together in Delft in a hurry. I ll put a copy there then.  https://github.com/gismo/gismo/blob/bca4e24e5addc7297c7708e9b94fde1b73d3cd0f/src/gsAssembler/gsAssembler.h#L463-L469